main.js

```
@@ -1,89 +1,54 @@
 const Apify = require('apify');
-const request = require('request-promise');
-const cheerio = require('cheerio');
-
-/**
- * Helper function which writes provided message into console log and then
- * exists the process with failure code.
- *
- * @param {String} errorMessage Message to be written into console log
- */
-function crash(errorMessage) {
-    console.log(errorMessage);
-    process.exit(1);
-}
+const { log } = Apify.utils;
 
 Apify.main(async () => {
-    // Load query from input
+    const proxyConfiguration = await Apify.createProxyConfiguration();
+    const requestQueue = await Apify.openRequestQueue();
     const input = await Apify.getValue('INPUT');
-    if (!input) crash('Key-value store does not contain INPUT.');
+    if (!input) log.error('Can not get input');
```

throw instead of log